grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionState.java

```diff
@@ -686,7 +756,7 @@ private void setResult(final T result) {
 
             this.result = result;
```

Comment:
Do we still want to assign result if we won't manage it?

Reply:
Yes. Unless I misunderstand the question. Not every exported type is required to be a liveness referent. So, setting the result is how we communicate (like a future) the result of the operation. The liveness referent is there to quickly mark the object graph as dead when the session expires or is closed. Non liveness referent exports won't get any form of notification when the session expires. If you truly have an export that doesn't have a result you will tend to be a "non-export" which technically exports a null value (after invoking the desired void() method).

Reply:
We discussed verbally. Question stands.